<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demo</class>
 <widget class="QWidget" name="Demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo</string>
  </property>
  <widget class="QPushButton" name="CalcButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>筛选</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="voltInput">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>12.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="voltLabel">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>0</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>电源电压/V</string>
   </property>
  </widget>
  <widget class="QLabel" name="currLabel">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>电流/A(限定最大值)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="currInput">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="condLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>筛选条件</string>
   </property>
  </widget>
  <widget class="QLabel" name="spdLabel">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>空速/(m/s)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="spdInput">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>7.0</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="thrustLabel">
      <property name="text">
       <string>静拉力(牛)&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="minStaThruInput">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="minEtaInput">
      <property name="text">
       <string>60</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="etaCondLabel">
      <property name="text">
       <string>空速下螺旋桨效率&gt;%</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dyThruLabel">
      <property name="text">
       <string>空速下动拉力(牛)&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="minDyThruInput">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="etaTolLabel">
      <property name="text">
       <string>空速下总效率&gt;%</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="etaTolInput">
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>电机KV值/(RPM/V)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="KV_Input">
      <property name="text">
       <string>220</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>电机内阻/Ω</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="r_Input">
      <property name="text">
       <string>.042</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="rtoLabel">
      <property name="text">
       <string>减速比1：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="gearEtaLabel">
      <property name="text">
       <string>减速组机械效率%</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="gearRtoInput">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="gearEtaInput">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>提示：
1、减速比大于1代表螺旋桨转速低于电机转速
2、计算过程可能会消耗20秒，请耐心等待，
不要重复点击“筛选”按钮！！！</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
